api/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM alpine:3.15.0
+
+RUN apk add --update --no-cache \
+  build-base \
+  nodejs-current \
+  curl \
+  npm \
+  tzdata \
+  libxslt \
+  bash \
+  jq    \
+  git
+
+WORKDIR /app
+
+COPY api api/
```

Comment:
I think we're going to have to refine this copy, so we only include useful things in the bundle.
I know I said yesterday that we should move the installation itself to grunt, but I think it's ok to keep here. 

So what I'm thinking is: 
1. We move `api/server.js` to `api/src/server.js`. 
2. We only copy (Do we need an additional "api" folder wrapper? Is it maybe cleaner to just copy to /?)
- `api/src/`
- `api/package.json`
- `api/package-lock.json`
- `api/README.md`
- `api/LICENSE`
This way, the image will omit unnecessary files (like test files!). 
3. Run `npm ci --production`
4. Run `npm dedupe`
5. entrypoint